packages/devextreme/js/ui/tree_view/ui.tree_view.base.js

```diff
@@ -577,11 +577,21 @@ const TreeViewBase = HierarchicalCollectionWidget.inherit({
         };
     },
 
-    _initMarkup: function() {
+    _initMarkup() {
         this._renderScrollableContainer();
         this._renderEmptyMessage(this._dataAdapter.getRootNodes());
+
         this.callBase();
-        this.setAria('role', 'tree');
+
+        this._setAriaRoleToElement();
+    },
+
+    _setAriaRoleToElement() {
```

Comment:
I think that suffix is excess. It's clear enough that we add a role to an element.
```suggestion
    _setAriaRole() {
```

Reply:
[17d788a031862755f5b9987fbe73de76e65e459c]